Formula/efl.rb

```diff
@@ -54,3 +58,27 @@ def post_install
     system bin/"eet", "-V"
   end
 end
+
+__END__
+diff --git a/src/lib/ecore_cocoa/ecore_cocoa_app.m b/src/lib/ecore_cocoa/ecore_cocoa_app.m
+index 8df1be1..e29b934 100644
+--- a/src/lib/ecore_cocoa/ecore_cocoa_app.m
++++ b/src/lib/ecore_cocoa/ecore_cocoa_app.m
+@@ -45,7 +45,7 @@ + (Ecore_Cocoa_Application *)sharedApplication
+ 
+ - (void)internalUpdate
+ {
+-   [_mainMenu update];
++   [[self mainMenu] update];
+    // FIXME Will not compile with GNUStep (member is named "_main_menu")
+ }
+ 
+@@ -76,7 +76,7 @@ - (void)run
+ {
+    [self finishLaunching];
+ 
+-   _running = 1;
++   [self setValue:[NSNumber numberWithShort:1] forKey:@"_running"];
```

Comment:
It might work, but ultimately I disagree with this.

The correct fix is to not rely on internal variables, create your own variable (which can also be called `_running`), and override `isRunning`.

```objc
- (BOOL)isRunning
{
    return _running;
}
```

Of course though, this should be submitted upstream and ideally reviewed.

Reply:
No rush, I too am doing this to check off a checkbox :P